src/constants.ts

```diff
@@ -1,8 +1,12 @@
 export const SAMPLE_DATE = new Date();
```

Comment:
so I see you have a file for example data
move every here, and divide them to objects according to topic

```ts
const SAMPLE_PROFILE_PICTURES = [
   ...
]
```

Reply:
```ts
EXAMPLE_TEXTS: {
    DESCRIPTION: 'bla bla',
    ...
}